doc/OpenMPSupport.md

```diff
@@ -0,0 +1,220 @@
+# TAFFO OpenMP Support
+TAFFO has preliminary support for [OpenMP](https://www.openmp.org) parallel constructs.
+The goals of the integration are the following:
+- The usage of a simple workflow to easily patch the runtime OpenMP function during optimizations,
+ requiring the least amount of work necessary for intermediary passes.
+- Basic support for the most common OpenMP directives, such as `omp parallel for`. 
+ This allows TAFFO to convert the floating-point logic inside the supported OpenMP pragmas to fixed-point.
+- Conservativeness in handling not supported directives, such as `omp task`, to avoid failures.
+- Evaluate the results with benchmarks adapted to OpenMP and containing floating-points.
+
+Note that a thorough analysis of the effects of data-dependency in the parallel execution is out of the scope of the integration.
+
+**If you are a TAFFO user and not interested in the technical details**, note the following:
```

Comment:
I suggest to re-word the sentence as in "More technical details for developers are available elsewhere...."

Reply:
Move to another document `elsewhere` implementation details.

Reply:
Split the old document in UserGuide, DevGuide, and benchmarks